devdox/app/services/git_tokens_service.py

```
@@ -130,122 +128,73 @@ def mask_token(token: str) -> str:
         self.crypto_store = crypto_store
+        self.git_manager = git_manager
 
     @classmethod
     def with_dependency(
-        cls, user_store: Annotated[TortoiseUserStore, Depends()],
-        crypto_store: Annotated[FernetEncryptionHelper, Depends(get_encryption_helper)],
+            cls,
+            user_store: Annotated[TortoiseUserStore, Depends()],
+            label_store: Annotated[TortoiseGitLabelStore, Depends()],
+            crypto_store: Annotated[FernetEncryptionHelper, Depends(get_encryption_helper)],
+		    git_manager: Annotated[RepoFetcher, Depends()]
```

Flake8 (7.2.0)

[error] indentation contains mixed spaces and tabs

(E101)